crypto/package.json

```diff
@@ -20,7 +20,7 @@
   "devDependencies": {
     "@babel/eslint-parser": "^7.23.3",
     "@types/chai": "^4.3.11",
-    "@types/mocha": "^10.0.4",
+    "@types/mocha": "^10.0.6",
```

Comment:
would suggest keeping mocha types version consistent across the repo as sometimes with different versions being installed I saw typedoc failng.